### ISSUE TITLE
Data Update not returning updated object (API v4.5.1)

### ISSUE THREAD

**Issue by alexandervalencia** [wedeploy/wedeploy-sdk-js#281]
### Environment
JavaScript SDK version: 4.5.1

Environment:
  Browser: Chrome 71.0.3578.98
  Node: v10.14.1
  React: v16.3.2

### Steps to Reproduce
Using WeDeploy.data().update(), update a row of a database for the project. Here's the example from the documentation:
```js
WeDeploy
  .data('https://<serviceID>-<projectID>.wedeploy.io')
  .update('movies/115992383516607958', {
    "rating": 9.1
  }).then(function(movie) {
    console.log(movie);
  });
```

### Expected Behavior

I get the updated object returned as `movie` in the `.then()` and would see in my console:
```json
{
  "movie": "Star Wars VIII: The Last Jedi",
  "rating": 9.1
}
```
### Actual Behavior
A blank string is being returned.
```js
movie => {
  console.log(typeof movie) // String
  console.log(movie) // logs out: ""
}
```

### Reproducible Demo

If needed I can show on my laptop (in LAX office) and share the repository.

**Comment by alexandervalencia** [comment]
Docs show the updated object should be returned: https://wedeploy.com/docs/data/updating-data/#1

**Comment by ipeychev** [comment]
Thanks for opening the issue!
There is no problem in SDK JS, the server doesn't return anything in case of PATCH requests. Will open an issue to see how easy that could be implemented and keep posting here the results.

**Comment by ipeychev** [comment]
@alexandervalencia Are you using Data serice on a project, deployed on wedeploy.com?

**Comment by alexandervalencia** [comment]
Yes, I'm using: `"image": "wedeploy/data:2.8.0"`

**Comment by anitaderiane** [comment]
Hey @ipeychev, is this still in progress? If not, can you please update the status of the ticket? 
Thank you!

**Comment by ipeychev** [comment]
The issue was fixed in Data.

**Comment by ipeychev** [comment]
This was fixed in Data, but a new image was not released. It is planned for the next week, so I will set this issue for the next milestone.

**Comment by monicaposin** [comment]
@ipeychev - Status of this is still in "In Progress" - Should it be in "Waiting for Release" status as per your last comment?

**Comment by ipeychev** [comment]
@monicaposin We are still waiting for a new version of Data which will solve the issue.
/cc @igr

**Comment by igr** [comment]
@monicaposin - to clarify, the new version of `data` that has this functionality is already released (both master and 2.x). however, for some reason, it can not be used in production; while locally it works. That issue is not related to this functionality.

See: https://github.com/wedeploy/engine/issues/1004

**Comment by ipeychev** [comment]
Fixed in `wedeploy/data:2.10.1`.

Example `wedeploy.json`:

```json
{
	"id": "data",
	"image": "wedeploy/data:2.10.1"
}
```
